app/src/main/java/ru/hse/spb/secretsanta/data/Wishlist.kt

```diff
@@ -0,0 +1,12 @@
+package ru.hse.spb.secretsanta.data
+
+/**
+ * Wishlist of a person. Contains two lists: one for wishes and another one for anti wishes. Each wish or anti
+ * wish is a [WishlistEntry].
+ */
+class Wishlist(val wishes: List<WishlistEntry>, val antiWishes: List<WishlistEntry>)
+
+/**
+ * One entry of a [Wishlist]
+ */
+inline class WishlistEntry(val description: String)
```

Comment:
I'm sure that this is the right usage of `inline class`. This class can potentially be extended. It semantically is not a wrapper, it just happened to have a single field at the moment. I'd say that `data class` would be better.